src/main/resources/db/migration/V11_6__add_column_dialogmotekandidat.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE PERSON_OVERSIKT_STATUS ADD COLUMN dialogmote_kandidat BOOLEAN DEFAULT null;
```

Comment:
Er dialogmøtekandidat egentlig ett ord? I så fall trenger man ikke understrek her, og det kan være `dialogmotekandidat` i stedet for `dialogmoteKandidat` i koden 👍

Reply:
Ja, du har rett i det. Vi behandler det som ett ord i `isdialogmotekandidat` i hvert fall. Endrer 👍